.github/CODEOWNERS

```
@@ -35,7 +35,7 @@ web/gui/ @jacekkolasa
 
 # Ownership by filetype (overwrites ownership by directory)
 *.am @Ferroin @tkatsoulas
-*.md @tkatsoulas
+*.md @tkatsoulas @andrewm4894
```

```suggestion
*.m?(d|dx) @tkatsoulas @andrewm4894
```

I am not sure we have any mdx outside of `docs/`, but I think it make sense to match bot md and mdx here.